CNV#30164: 4.15 CNV Networking release notes (#71842)
openshift/openshift-docs: virt/release_notes/virt-4-15-release-notes.adoc

[id="virt-4-15-networking"]
=== Networking

//CNV-30164 and CNV-30167: NIC hot plug feature is now GA. This was TP for 4.14. Moved the release note from the TP section to here. Covers both bridge and SR-IOV NICs

> * You can xref:../../virt/vm_networking/virt-hot-plugging-network-interfaces.adoc#virt-hot-plugging-network-interfaces[hot plug a secondary network interface] to a running VM. Hot plugging and hot unplugging is supported only for VMs created with {VirtProductName} 4.14 or later. Hot unplugging is not supported for Single Root I/O Virtualization (SR-IOV) interfaces.

Reviewer: Suggestion to spell out Virtual Machine (especially on the first mention in this section so far):
```suggestion
* You can xref:../../virt/vm_networking/virt-hot-plugging-network-interfaces.adoc#virt-hot-plugging-network-interfaces[hot plug a secondary network interface] to a running virtual machine (VM). Hot plugging and hot unplugging is supported only for VMs created with {VirtProductName} 4.14 or later. Hot unplugging is not supported for Single Root I/O Virtualization (SR-IOV) interfaces.
```

//CNV-13680: Connecting OVN-Kubernetes to underlying physical networks. Remove known issue BZ-1885605 after adding this note.
* {VirtProductName} now supports the localnet topology for xref:../../virt/vm_networking/virt-connecting-vm-to-ovn-secondary-network.adoc#virt-connecting-vm-to-ovn-secondary-network[OVN-Kubernetes secondary networks]. A localnet topology connects the secondary network to the physical underlay. This enables both east-west cluster traffic and access to services running outside the cluster, but it requires additional configuration of the underlying Open vSwitch (OVS) system on cluster nodes.

//CNV-30170: NetworkPolicies filtering using IP blocks with OVN-Kubernetes

> * An OVN-Kubernetes secondary network is compatible with the xref:../../networking/multiple_networks/configuring-additional-network.adoc#compatibility-with-multi-network-policy_configuring-additional-network[multi-network policy API] which provides the `MultiNetworkPolicy` custom resource definition (CRD) to control traffic flow to and from VMs. You can use the `ipBlock` attribute to define network policy ingress and egress rules for specific CIDR blocks.

Reviewer: Suggestion - I think a comma here might make the long sentence easier to read, up to you:
```suggestion
* An OVN-Kubernetes secondary network is compatible with the xref:../../networking/multiple_networks/configuring-additional-network.adoc#compatibility-with-multi-network-policy_configuring-additional-network[multi-network policy API], which provides the `MultiNetworkPolicy` custom resource definition (CRD) to control traffic flow to and from VMs. You can use the `ipBlock` attribute to define network policy ingress and egress rules for specific CIDR blocks.
```

> * An OVN-Kubernetes secondary network is compatible with the xref:../../networking/multiple_networks/configuring-additional-network.adoc#compatibility-with-multi-network-policy_configuring-additional-network[multi-network policy API] which provides the `MultiNetworkPolicy` custom resource definition (CRD) to control traffic flow to and from VMs. You can use the `ipBlock` attribute to define network policy ingress and egress rules for a specific CIDR block.

Reviewer 2: ```suggestion
* An OVN-Kubernetes secondary network is compatible with the xref:../../networking/multiple_networks/configuring-additional-network.adoc#compatibility-with-multi-network-policy_configuring-additional-network[multi-network policy API] which provides the `MultiNetworkPolicy` custom resource definition (CRD) to control traffic flow to and from VMs. You can use the `ipBlock` attribute to define network policy ingress and egress rules for specific CIDR blocks.
```
